tests/_support/AcceptanceTester.php

```diff
@@ -321,4 +321,20 @@ public function iConfirmTo($message)
         $this->seeInPopup($message);
         $this->acceptPopup();
     }
-}
+
+    /**
+     * @Then I should see an option :option
+     */
+    public function iShouldSeeAnOption($option)
```

Comment:
When reading the name of this function, you might think that it looks for a HTML option tag on the web page. Since it is actually being used to check that an input button element is present, maybe it would be better to rename the function to something like `iShouldSeeAnInputButton`? You would need to update the @step string too.